File: src/Java/src/LinkedNode.java
public class LinkedNode extends Node {
  private LinkedNode _prevLinkedNode;
  private LinkedNode _nextLinkedNode;

  public LinkedNode(int data) {
    super(data);
    prevLinkedNode(null);
    nextLinkedNode(null);
  }

  public void prevLinkedNode(LinkedNode newPrevLinkedNode) {
    _prevLinkedNode = newPrevLinkedNode;
  }

  public LinkedNode prevLinkedNode() {
    return _prevLinkedNode;
  }

  public void nextLinkedNode(LinkedNode newNexLinkedNode) {
    _nextLinkedNode = newNexLinkedNode;
  }

  public LinkedNode nextLinkedNode() {
    return _nextLinkedNode;
  }
}

File: src/JavaScript/src/LinkedNode.js
import Node from "./Node.js";

export default class LinkedNode extends Node {
  constructor(data) {
    super(data);

    this._prevLinkedNode = null;
    this._nextLinkedNode = null;
  }

  set prevLinkedNode(linkedNode) {
    this._prevLinkedNode = linkedNode;
  }

  get prevLinkedNode() {
    return this._prevLinkedNode;
  }

  set nextLinkedNode(linkedNode) {
    this._nextLinkedNode = linkedNode;
  }

  get nextLinkedNode() {
    return this._nextLinkedNode;
  }
}
